src/attributes/attr.js

```diff
@@ -3,11 +3,14 @@ define([
 	"../core/access",
 	"./support",
 	"../var/rnotwhite",
+	"./val",
```

Comment:
This existed in the code before my previous commit. See https://github.com/jquery/jquery/blob/0de798d6c2277a286c90e08de669a23e6e12a4a2/src/attributes/attr.js#L6